### PR TITLE
Update project documentation

### PR DESCRIPTION
**Pull request by jordigarcl**
This PR would close issue #2 .

I've made an attempt to update the README of the project to among other things, reflect new features and give some context about installation and contribution.

What do you guys think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+![Project Banner](https://github.com/PikaDude/Animal-Crossing-Music-Extension/blob/master/docs/banner.png)
+
+# Animal Crossing Music (Revamp) 🍀
```
Comment (jordigarcl) [inline README.md]: I propose to call this a 'Revamp' for the time being but I'm not sold on the name. I still hope we can transfer all these changes into the original project!

Comment (PikaDude) [reply README.md]: Maybe "Revived"?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+- Google Drive: [Animal Crossing Music Extension](https://drive.google.com/drive/folders/0B79uF1ZqAHtbN0l4eFR1NU9CMGc)
+
+In order to run this project locally, you will need to download the above directory and extract it under `./sounds`.
+
+## Contribute
+Contributions to the project are greatly appreciated!
+
+If you have a feature request, a proposal for improvement, want to report a bug or just to ask a question, feel free to open a public [issue](https://github.com/PikaDude/Animal-Crossing-Music-Extension/issues) on the project repository.
+
+[PRs](https://github.com/PikaDude/Animal-Crossing-Music-Extension/pulls) are equally welcomed. Just make sure to mention a relevant issue in its description!
```
Comment (jordigarcl) [inline README.md:61]: This will have to be updated if we implement #4

File: README.md (hunk 1 above)
Comment (jordigarcl) [reply README.md]: I will change it to "Revived" for now 👍

**Pull request merged**